docs/examples/zoomable-circle-packing.vg.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://vega.github.io/schema/vega/v5.json",
   "description": "An example of a zoomable circle packing layout for hierarchical data.",
-  "width": 800,
-  "height": 800,
+  "width": 600,
```

Comment:
Do we need to rebuild the image?

Reply:
Not sure I follow. I'm assuming that you're not referring to the thumbnail image, correct?

Reply:
Yes

Reply:
With the only spec update being the width and height (same aspect ratio), the existing thumbnail looks good in my dev environment and should do the trick.